data-pipeline/docker-compose.yml

```diff
@@ -59,9 +59,16 @@ services:
 
   http-materializer:
     image: zhenik/http-materializer:data-pipeline
-    build: http-materializer
     environment:
       KAFKA_BOOTSTRAP_SERVERS: kafka:9092
       SCHEMA_REGISTRY_URL: http://schema-registry:8081
       SOURCE_TOPIC: events-message-v1
-      DATABASE_REST_SERVICE_URL: http://db-mock:80/messages
\ No newline at end of file
+      DATABASE_REST_SERVICE_URL: http://db-mock:80/messages
+    restart: always
+    depends_on:
+      - zookeeper
+      - kafka
+      - db-mock
+      - kafka-setup
+      - http-producer
```

Comment:
sure we need this dependency?

Reply:
`db-mock` and `kafka` are two dependencies for `http-materialiser`,  `kafka-setup` is [found approach](https://github.com/confluentinc/cp-docker-images/issues/694) how to create topics in docker-compose files at container start-up. 
I can agree that `http-producer` does not need to be here

Reply:
I meant `http-producer` only. Github comments go to the last line.